app/views/ama_layout/_siteheader.html.erb

```diff
@@ -4,36 +4,30 @@
       <li class="name">
         <%= link_to(image_tag("AMA-Logo.png"), "https://www.ama.ab.ca") %>
       </li>
-       <!-- Remove the class "menu-icon" to get rid of menu icon. Take out "Menu" to just have icon alone -->
       <li class="toggle-topbar menu-icon"><a href="#"><span></span></a></li>
     </ul>
     <section class="top-bar-section show-for-small-only">
       <ul class="right">
-        <%= render partial: "ama_layout/main_top_nav_item", collection: navigation, as: :nav_item if navigation %>
+        <%= render partial: "ama_layout/main_top_nav_item", collection: navigation.items, as: :nav_item %>
         <li class="divider"></li>
-        <li class="show-for-small-only"><a href="<%= Rails.configuration.youraccount_site %>/help#faq1">Help</a></li>
-        <li class="divider show-for-small-only"></li>
-        <li class="show-for-small-only"><a href="<%= Rails.configuration.amaabca_site %>/membership/contact-us--centre-locations-hours-and-contact-information">Contact Us</a></li>
+        <li><a href="<%= Rails.configuration.amaabca_site %>">AMA Website</a></li>
```

Comment:
I thought about it but decided against it.

My rationale:

This is a Rails engine, so Rails.configuration will always be available.
Because it's a Rails engine, I felt that adding a configuration class and supporting initializers in each app was was adding more duplication of effort than the value it provided.